typescript/src/schema/proto.ts

```diff
@@ -382,6 +393,10 @@ function augmentMethod(
   if (method.methodConfig.timeout) {
     method.timeoutMillis = milliseconds(method.methodConfig.timeout);
   }
+  if (method.options && method.options['.google.api.http']) {
+    const httpRule = method.options['.google.api.http'];
+    method.headerRequestParams = getHeaderParms(httpRule);
```

Comment:
it means that if a method does not have `.google.api.http` annotation the `.headerRequestParam` will be `undefined`, right? Let's explicitly assign it to `[]`.

Reply:
That makes sense to me, since it will check the length of `headerRequestParam` array